build-logic/build.gradle

```diff
@@ -0,0 +1,12 @@
+plugins {
```

Comment:
why are you renaming to `build-logic`? We should be consistent in every module and we use `buildSrc` everywhere.

Reply:
I thought that now you use build-logic everywhere, because that’s what they did in micronaut-openapi. I thought that everything needed to be brought to one form. I don’t know why in micronaut-openapi this folder was renamed to build-logic

Reply:
Please, use `buildSrc` in this Pr. Thanks.